src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -58,16 +58,42 @@ def __iter__(self):
 
 
 def create_exists_filter_group(term):
+    """Creates a :class`FilterGroup` to find events where filter data exists.
```

Comment:
These dont mention that theyre useful for when py42 does not yet support the term like the others do